agenta-cli/agenta/sdk/litellm/litellm.py

```diff
@@ -89,16 +89,20 @@ def log_stream_event(
                 log.error("LiteLLM callback error: span not found.")
                 return
 
-            result = kwargs.get("complete_streaming_response")
```

Comment:
It does not look like you can remove this. It seems they get the result from there in the case of streaming, no?